metadata-ingestion/src/datahub/ingestion/source/tableau/tableau.py

```diff
@@ -397,7 +399,7 @@ class TableauConfig(
     )
 
     page_size: int = Field(
```

Comment:
The only remaining usage of `page_size` is here
https://github.com/datahub-project/datahub/blob/3108b539d0365cfa1477e89e04447430983434d2/metadata-ingestion/src/datahub/ingestion/source/tableau/tableau.py#L996-L998

We can get rid of it in favour of fine-grained `database_servers_page_size`. If so we can deprecate `page_size` and make `page_size_override` required param in `get_connection_objects` method.

Alternatively, we can left `page_size` for that remaining usage and others in the future.